packages/shared/src/porter.ts

```diff
@@ -13,9 +13,8 @@ import { Base64EncodedBytes, ChecksumAddress, HexEncodedBytes } from './types';
 import { fromBase64, fromHexString, toBase64, toHexString } from './utils';
 
 const porterUri: Record<string, string> = {
-  mainnet: 'https://porter.nucypher.community',
-  tapir: 'https://porter-tapir.nucypher.community',
-  oryx: 'https://porter-oryx.nucypher.community',
+  mainnet: 'https://porter.nucypher.io',
+  tapir: 'https://porter-tapir.nucypher.io',
   lynx: 'https://porter-lynx.nucypher.community',
```

Comment:
If we are switching, should we just switch all? i.e. also switch lynx - cc @KPrasch

Reply:
Yes, but currently https://porter-lynx.nucypher.io/ doesn't work

Reply:
Yep. I just mean that we shouldn't merge this before making that happen.